### PR TITLE
Change log for 0.11.0

### PR DESCRIPTION
**Pull request by mathieuleclaire**

### CODE REVIEW

**Comment by cquiroz** [comment]
PR #220 and #221 are the same thing. They could be merged onto a single line

File: manual/src/ornate/changelog.md [hunk 1]
```diff
@@ -1,5 +1,21 @@
...
+> 2018 March 15
+
+- [#228](https://github.com/scalacenter/scalajs-bundler/pull/228): Implements npmUdpdate in a separate object;
+- [#225](https://github.com/scalacenter/scalajs-bundler/pull/225): Add --mutex to yarn command;
+- [#222](https://github.com/scalacenter/scalajs-bundler/pull/222): Make library example work in OSX;
+- [#220](https://github.com/scalacenter/scalajs-bundler/pull/220): Support passing an extra list of arguments to webpack;
+- [#218](https://github.com/scalacenter/scalajs-bundler/pull/218): Fix #136: Add more precise jsdom detection:
+- [#216](https://github.com/scalacenter/scalajs-bundler/pull/216): Fix #200: current webpack devserver version does not accept watchOptions;
+- [#215](https://github.com/scalacenter/scalajs-bundler/pull/215): Fix #99: Relax NPM dependency version conflict resolution;
+- [#213](https://github.com/scalacenter/scalajs-bundler/pull/213): Fix #168: Update the snabbdom facade for Scala.js 1.0
```
Comment (julienrf) [inline manual/src/ornate/changelog.md]: Unfortunately, on the website the `#XXX` numbers will not be turned into links. You should either remove them or make them actual markdown links.

Comment (mathieuleclaire) [reply manual/src/ornate/changelog.md]: I don't understand, I used the same syntax as the one already used in the changelog.md.

Comment (julienrf) [reply manual/src/ornate/changelog.md]: I’m talking about the `Fix #168`, for instance (not the `[#213](…)`)

Comment (mathieuleclaire) [reply manual/src/ornate/changelog.md]: OK, I fix this.

**Comment by julienrf** [comment]
This is great, thanks @mathieuleclaire!

**Pull request merged**